types/html-validator/index.d.ts

```diff
@@ -0,0 +1,76 @@
+// Type definitions for html-validator 4.0
+// Project: https://github.com/zrrrzzt/html-validator
+// Definitions by: Takesi Tokugawa <https://github.com/TokugawaTakesi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+/**
+ * If desired validation format is 'json', really not json, JavaScript object (parsed object will be returned)
+ */
+declare function HtmlValidator(
+    options:
+        HtmlValidator.OptionsForHtmlFileAsValidationTargetAndObjectAsResults |
```

Comment:
If you don't like long names like this, please make a suggest. 
If make it shorter, unambiguity will be lost.